src/init/prepare.sh

```diff
@@ -52,6 +52,9 @@ getEnv() {
     # allow limiting log pollution on startup
     silent_init=${SILENT_INIT:-false}
     dev_mode=${DEV_MODE:-false}
+    auto_migration=${AUTO_MIGRATION:-false}
```

Comment:
I would call this `auto_db_update` so it is reminiscent of the `db:update` command